packages/auth-client/src/cookie-helpers.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+import Cookies from 'js-cookie'
+
+import createDebug from 'debug'
+
+const debug = createDebug('origin:auth-client:')
+
+function loadTokenCookie(walletAddress) {
+  const key = 'authtoken:' + walletAddress
+  const tokenData = Cookies.getJSON(key)
+
+  debug('Found token data in cookies', key, tokenData)
+
+  return tokenData
+}
+
+function saveTokenCookie(walletAddress, tokenData) {
+  const key = 'authtoken:' + walletAddress
+  debug('Saving token data to cookies', key, tokenData)
+  Cookies.set(key, tokenData, {
```

Comment:
Sure. Will do. As of now, this file isn't used. But, it will be used once I start with the changes on the DApp side.

Reply:
If the intention behind using cookie storage is to avoid cookies being stolen when hosted on IPFS, we should also set the path options so that the cookie is valid only on the current page and below. Otherwise, by default, js-cookie cookies are available to the entire domain.

I think it should be set to `path: ''`

Reply:
Have done this. But will have ensure that `path: ''` works